test/v4/createAmendAnnualSubmission/def1/Def1_CreateAmendAnnualSubmissionValidatorSpec.scala

```diff
@@ -178,13 +177,9 @@ class Def1_CreateAmendAnnualSubmissionValidatorSpec
   private def validator(nino: String, businessId: String, taxYear: String, body: JsValue) =
     validatorFactory.validator(nino, businessId, taxYear, body)
 
```

Comment:
Please remove `with MockSharedAppConfig with MockSeBusinessFeatureSwitches` as it is not required, thanks.